src/OrchardCore/OrchardCore.Data.Abstractions/DatabaseHelper.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace OrchardCore.Data;
+public class DatabaseHelper
+{
+    public static string GetStandardPrefix(string prefix)
```

Comment:
Keep this in the implementation project, and make it internal. It's only used in this project.

Reply:
done.